controller-server/src/main/java/com/yahoo/vespa/hosted/controller/LockedApplication.java

```diff
@@ -54,7 +54,7 @@ private LockedApplication(Builder builder) {
               builder.outstandingChange, builder.ownershipIssueId, builder.metrics, builder.rotation);
     }
 
-    public LockedApplication withProjectId(long projectId) {
+    public LockedApplication withProjectId(Optional<Long> projectId) {
```

Comment:
There's an `OptionalLong` that's would be better suited here (less boxing).

Reply:
Sure!